wherehows-web/app/components/datasets/owners/suggested-owners.ts

```diff
@@ -0,0 +1,43 @@
+import Component from '@ember/component';
+import { IOwner } from 'wherehows-web/typings/api/datasets/owners';
+import { computed, get } from '@ember/object';
+import ComputedProperty, { empty } from '@ember/object/computed';
+
+export default class DatasetsOwnersSuggestedOwners extends Component {
+  /**
+   * Sets the class names binded to the html element generated by this component
+   */
+  classNames = ['dataset-authors-suggested'];
+
+  /**
+   * Whether or not the component is expanded. If not, users will only see the initial header information
+   * whereas if expanded then users will see the list of all suggested owners
+   * @type {boolean}
+   * @default false
+   */
+  isExpanded = false;
+
+  /**
+   * Computed based on the owners array, detects whether this array is empty or not
+   * @type {ComputedProperty<boolean>}
+   */
+  isEmpty = empty('owners');
+
+  /**
+   * Passed in value from parent component, `dataset-authors`, a.k.a. systemGeneratedOwners, this list
+   * represents a possible list of owners provided by scanning various systems.
+   * @type {Array<IOwner>}
+   * @default []
+   */
+  owners: Array<IOwner> = this.owners || [];
```

Comment:
Please move this (no pun intended) to the constructor, although it works now it's a bug that will be fixed in a future version of TS. If you look at the emit in JS you'll see why it doesn't really make sense. thanks

Reply:
Ah I've been lied to by blogs. Will make that change.